docs/client/concepts/parameter-stores.mdx

```diff
@@ -7,30 +7,95 @@ slug: /client/concepts/parameter-stores
 
-## What is a Parameter Store?
-Rather than thinking in terms of Statsig entities (Feature Gates, Experiments, Layers, Dynamic Configs), Parameter Stores allow you to think in terms of *parameters* - things in your app that you want to be configurable remotely.  Parameters decouple your code from the configuration, much like Statsig Layers decouple your code from the specific experiment.  It is a level of indirection that allows you to run any set of experiments, or change gating or values on the fly, without hardcoding an experiment name in your app.  Each of these parameters that you define and check can be remapped at will between statsig entities - provided you maintain the same type for that parameter.  As a result, the parameters will receive different values depending on the `StatsigUser` you initialize the SDK with.  These dynamic values can be updated at any time, without a mobile code change, and without waiting for your mobile release cycle.
+As a developer, hardcoded configuration values in mobile apps can become a bottleneck. Whether you're tweaking onboarding flows, changing UI elements, or managing feature rollouts, constantly updating your codebase and waiting for release cycles isn't efficient. **Parameter Stores** offer a flexible, real-time approach to managing configurations directly through the Statsig Console.
 
-## How should I use Parameter Stores?
-Here is a suggested workflow:
-1. Create a parameter store for you team or project.  Parameter Stores are an object to hold related parameters.
-2. As you are adding to your mobile app, consider which variables you'd like to be controlled on the server-side, rather than hard coded into your app. These can be things like:
+---
+
+### **What Exactly is a Parameter Store?**
+
+Parameter Stores let you manage the parameters driving features in your app (boolean flags, text resources, numerical settings) without hardcoding them into your code. This allows you to dynamically adjust app behavior by decoupling configuration from code.
+
+Unlike Feature Gates, Experiments, or Dynamic Configs tied to specific workflows, Parameter Stores offer the flexibility to define and manage parameters independently, which can be remapped to these features later.
+
+---
+
+### **How to Use Parameter Stores: A Practical Example**
+
+Let’s say you want to implement a new onboarding flow, but only for users on the latest version of your app. Here’s how Parameter Stores help without needing constant code updates:
+
+1. **Create a Boolean Parameter**
+
+   In the Statsig Console, create a boolean parameter called `enable_new_onboarding` and set its default value to `false`. This will be your feature toggle.
+
+2. **Use the Parameter in Your Code**
+
+   Fetch the value of the parameter in your app using the Statsig SDK:
+
+   ```java
+   boolean newOnboardingEnabled = StatsigClient.getConfig("enable_new_onboarding").getBoolean("value", false);
```

Comment:
its not getconfig, its getParameterStore("store_name").getBoolean("enable_new_onboarding")